jetty-client/src/test/java/org/eclipse/jetty/client/http/HttpReceiverOverHTTPTest.java

```diff
@@ -330,15 +330,19 @@ public boolean headerComplete()
                     {
                         throw new RuntimeException(e);
                     }
+                    return true;
                 }
-                return true;
-            }
-        };
+            };
 
-        // start thread 1 receive() call from here
-        httpReceiverOverHTTP.receive();
+            // start thread 1 receive() call from here
+            httpReceiverOverHTTP.receive();
 
-        // make sure no exception was reported
-        assertThat(failureRef.get(), nullValue());
+            // make sure no exception was reported
+            assertThat(failureRef.get(), nullValue());
+        }
+        finally
+        {
+            Log.getLogger(HttpReceiver.class).setDebugEnabled(false);
```

Comment:
This undoes a setting from `jetty-logging.properties` (if being used to debug other things in the test cases).
Get the value before you test, set it to true, and restore the value.

Reply:
That's reasonable, let me do that.